### ISSUE TITLE
Replicate kannikkiy's script translator.

### ISSUE THREAD

**Issue by lodici** [magarena/magarena#539]
see [forum for exe](http://www.slightlymagic.net/forum/viewtopic.php?f=82&t=17928&p=189537#p189532).

![screenshot00383](https://cloud.githubusercontent.com/assets/2443519/11546073/e5170482-9942-11e5-88a2-6de9f0b7355c.png)

To do something similar in Magarena but more efficiently, add a language selector to the download dialog, then...

```
For each missing image
- read download link from script file.
- if non-english selected, substitute "en" in download link with language code.
- check if download URL exists and attempt to download image.
- if link missing or download fails (suppress warning), download english version as normal
```

**Comment by lodici** [comment]
The problem with a simple replacement of the language code is that there is likely to be a lot of misses. For example, [Goblin Offensive](http://magiccards.info/pch/en/56.html) has an ``image`` property of ``http://magiccards.info/scans/en/pch/56.jpg``. Replacing the language code will generate a miss because there are no other languages available for the ``pch`` set. However, there are images available in other languages from other sets, for example there is a [German version](http://magiccards.info/scans/de/us/192.jpg) from Urza's Saga (``http://magiccards.info/scans/de/us/192.jpg``).

One solution could be to update scripts builder to include a list of all ``mtgimage.com`` set codes from the json feed in which a card is found in each script (``sets=``). These codes can then be mapped to the equivalent ``magiccards.info`` set codes. Then during download, can try each set for a given language until an image is found or not.

**Comment by lodici** [comment]
Actually that won't work either because the image filename is also different, ``56.jpg`` for English and ``192.jpg`` for German.

**Comment by lodici** [comment]
Another more flexible but also manual way is to have language equivalent versions of the [``magarena-scripts-builder/INPUT/CardImages.txt``](https://github.com/magarena/magarena-scripts-builder/blob/master/INPUT/CardImages.txt) file. Each would have its own maintainer(s). It would require an update to the script file so that multiple image links can be stored.

**Comment by lodici** [comment]
Implemented as described in #558.

**Comment by lodici** [comment]
Implemented as described in #558.
